packages/merge-styles/src/Stylesheet.ts

```diff
@@ -50,16 +50,16 @@ let _stylesheet: Stylesheet;
  * @public
  */
 export class Stylesheet {
-  private _styleElement: HTMLStyleElement;
-  private _rules: string[];
+  private _styleElement!: HTMLStyleElement;
+  private _rules!: string[];
   private _config: IStyleSheetConfig;
-  private _rulesToInsert: string[];
-  private _timerId: number;
-  private _counter: number;
-  private _keyToClassName: { [key: string]: string };
+  private _rulesToInsert!: string[];
+  private _timerId!: number;
+  private _counter!: number;
+  private _keyToClassName!: { [key: string]: string };
 
   // tslint:disable-next-line:no-any
-  private _classNameToArgs: { [key: string]: { args: any, rules: string[] } };
+  private _classNameToArgs!: { [key: string]: { args: any, rules: string[] } };
```

Comment:
not sure why these were needed.

Reply:
Strict initialization check is now enabled